File: python/11_coin_piles.py
n = int(input().strip())

for _ in range(n):
    a, b = map(int, input().strip().split())

    if (a + b) % 3 == 0 and min(a, b) * 2 >= max(a, b):
        print("YES")
    else:
        print("NO")

File: cpp/11_coin_piles.cpp
#include <iostream>

using namespace std;

int main() {
    int n;
    cin >> n;

    while (n--) {
        int a, b;
        cin >> a >> b;

        if ((a + b) % 3 == 0 && min(a, b) * 2 >= max(a, b)) {
            cout << "YES" << endl;
        } else {
            cout << "NO" << endl;
        }
    }

    return 0;
}
